server/adapters/prismic.js

```diff
@@ -324,91 +338,107 @@ const getDataForApi = function ( req, listener ) {
  * Load data for Page response.
  *
  */
-const getDataForPage = function ( req, listener ) {
-    return new Promise(( resolve, reject ) => {
+const getDataForPage = function (req, listener) {
+    return new Promise((resolve, reject) => {
         const data = {
             item: null,
             items: null
         };
-        const doQuery = function ( type, uid ) {
+        const doQuery = function (type, uid) {
+            console.log(cache.api);
             let query = [];
-            const navi = getNavi( type );
-            const form = getForm( req, cache.api, type );
-            const isNaviNoForm = (navi && !cache.api.data.forms[ type ]);
-            const done = function ( json ) {
-                if ( !json.results.length ) {
+            const navi = getNavi(type);
+            // const form = getForm(req, cache.api, type);
+            // const isNaviNoForm = (navi && !cache.api.data.forms[type]);
+            const isNaviNoForm = navi && type === "studio";
```

Comment:
As far as I can tell `studio` is the only page that actually needs this to work so just hardcoded it here 🤔